awscli/customizations/wizard/wizards/iam/new-role.yml

```diff
@@ -174,7 +174,9 @@ plan:
               Type: "AWS::IAM::Role"
               Properties:
                 RoleName: {role_name}
+                {% if {role_description} !=   %}
```

Comment:
I'm curious if there's a more idiomatic way to specify `role_description` being null. Does `!= ''` satisfy the conditional? That may look more complete if it also applies to `null` values correctly.

Reply:
Hi @nateprewitt I agree that looks off but neither `%if {role_description} != '' %}` or `{% if {role_description} %}` worked. And it looks like either a `==` or `!=` operator is required [here](https://github.com/aws/aws-cli/blob/v2/awscli/customizations/wizard/core.py#L204-L232).

But adding quotes around role_description like this worked:  `{%if '{role_description}' != '' %}`, so I updated the PR. Thanks!